package.json

```diff
@@ -54,7 +61,7 @@
   ],
   "scripts": {
     "build": "tsup",
-    "type-check": "tsc --noEmit",
+    "type-check": "tsc --skipLibCheck --noEmit",
```

Comment:
why do we need to add skipLibCheck here?

Reply:
it was failing on a bunch of declaration files in the ai-jsx node_modules folder

Reply:
which i didn't really understand since tsconfig excludes node_modules

Reply:
Looks like excludes doesn't actually exclude files from type checking
https://www.typescriptlang.org/tsconfig#exclude

Reply:
I think using skipLibCheck should be fine though